src/main/java/io/openliberty/tools/intellij/util/LibertyProjectUtil.java

```diff
@@ -18,6 +18,7 @@
 import com.intellij.psi.search.FilenameIndex;
 import com.intellij.psi.search.GlobalSearchScope;
 import com.intellij.terminal.JBTerminalWidget;
+import com.intellij.terminal.ui.TerminalWidget;
```

Comment:
Do we need this import `TerminalWidget`? and similarly `TerminalView` in line 30

Reply:
Yes we need `TerminalWidget`, we are using this in Line no.242.
Removed TerminalView from imports.